internal/rbd/rbd_attach.go

```diff
@@ -91,25 +90,43 @@ var (
 	}
 )
 
+type deviceInfo interface {
+	GetName() string
+	GetPool() string
+	GetRadosNamespace() string
+	GetDevice() string
+}
+
 // rbdDeviceInfo strongly typed JSON spec for rbd device list output (of type krbd).
 type rbdDeviceInfo struct {
```

Comment:
We no longer have to save ID ? 
It's not used anywhere ?

Reply:
Indeed, it is not used anywhere for attaching/mounting.